aioxmpp/carbons/xso.py

```diff
@@ -0,0 +1,103 @@
+########################################################################
+# File name: xso.py
+# This file is part of: aioxmpp
+#
+# LICENSE
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program.  If not, see
+# <http://www.gnu.org/licenses/>.
+#
+########################################################################
+import aioxmpp.xso as xso
+
+from aioxmpp.utils import namespaces
+
+from ..misc import Forwarded
+from ..stanza import Message
+
+
+namespaces.xep0280_carbons_2 = "urn:xmpp:carbons:2"
+
+
+class Enable(xso.XSO):
```

Comment:
Enable and Disable should be registered as IQ payload. Even if they are never received as IQ payloads, they should be registered for documentation purposes. (Perhaps we want a decorator that registers for documentation and future type checking purposes only, and does not enable receiving those stanzas).